### PR TITLE
Added some controls to the chart

### PR DESCRIPTION
**Pull request by eldang**
1. The code diffs will be a lot easier to read if you merge https://github.com/coregis/ryht-charter-campuses/pull/8 first
2. There's a good chance you'll want to tweak the formatting of the control links.  That should be safe to do as long as you keep the padding in pixels, because the JS will look up that value and use it to calculate sizes for various elements.
3. Currently the controls div has a load of empty space in it - I anticipate putting the UI for choosing variables in there.

### CODE REVIEW

File: scripts/functions.js [hunk 1]
```diff
@@ -0,0 +1,500 @@
+// functions.js defines functions that will be called later
+
+
+// global variable for the path to the historical districts data file
+var districtsFile = 'data/qrySumStatsAllDistAllYears.csv';
+// data structure to hold state for the chart; the actual data will be attached on load
+// see https://github.com/d3/d3-format#locale_format for tick format strings
+var chartData = {
+	svgID: 'chart',
+	visible: true,
```
Comment (eldang) [inline scripts/functions.js:10]: If you want the chart to be folded away when the page first loads, just set this to `false`.

Comment (coregis) [reply scripts/functions.js:10]: This is really helpful. Is the var districtsFile the only place where csv needs to be specified? So when we swap in updated data, if we choose to change file name, we just change it here and charts will still work?

Comment (eldang) [reply scripts/functions.js:10]: That's correct, as long as the variable names are consistent.

**Review by coregis — APPROVED** [review]
I like these chart control-related changes. One thing we might consider for Hide Chart in next iteration is to add an 'x', radio button, or something else...it's probably intuitive for some people to click the text, but might be more intuitive for others if we add a subtle graphic cue.

**Comment by eldang** [comment]
> I like these chart control-related changes. One thing we might consider for Hide Chart in next iteration is to add an 'x', radio button, or something else...it's probably intuitive for some people to click the text, but might be more intuitive for others if we add a subtle graphic cue.

Good idea.  It should be pretty quick for me to add something like the close button we used to have on the sidenav.

**Pull request merged**